test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -0,0 +1,75 @@
+kind: StatefulSet
+apiVersion: apps/v1beta1
+metadata:
+  name: csi-gce-controller
+spec:
+  serviceName: "csi-gce-pd"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-gce-pd-driver
+  template:
+    metadata:
+      labels:
+        app: csi-gce-pd-driver
+    spec:
+      serviceAccount: csi-gce-pd
+      containers:
+        - name: csi-external-provisioner
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-provisioner:v0.2.0
+          args:
+            - "--v=5"
+            - "--provisioner=csi-gce-pd"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-attacher
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-attacher:v0.2.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: gce-driver
+          securityContext:
+            privileged: true
```

Comment:
Does the Stateful set actually need to be privileged?  It should not have anything reaching in to the host machine.

Reply:
no, fixed